addons/mail/static/src/core/common/chat_window.scss

```diff
@@ -1,3 +1,8 @@
+body:has(.o-mail-ChatWindow.o-mobile) {
+    // Disable text-highlighting/magnifier on safari mobile to solve conflicts with long press.
+    -webkit-user-select: none;
+}
+
```

Comment:
I don't think this is worth it: long press on message seems to work fine without it, long press outside sometimes we may want the default behavior of OS.

Besides this rule doesn't take into account some other contexts, e.g. long press on messages in mailboxes (which is not inside a chat window).